### ISSUE TITLE
Trello Problem

### ISSUE THREAD

**Issue by chrisfrees** [ovk/brackets-simple-todo#15]
I'm having trouble with the Trello integration. I connected my account correctly but I get the error: 

**Failed to read list of to-do items**
**Failed to get cards. Response: 401 - Unauthorized**

**Comment by ovk** [comment]
Hi @chrisfrees. Did you choose your board and list successfully in the Simple To-Do settings for Trello? And if yes - are you the owner of the board or is it shared by someone else?

**Comment by chrisfrees** [comment]
Yes I was able to view my Trello boards and lists and I am the owner of the board.

**Comment by ovk** [comment]
Could you please tell your operating system name and what Brackets version are you using?

Also please try to select a different board/list (for example, the default Welcome Board) to see if the error persists. 

Another thing to check is go to your Trello account settings and make sure that there is _Brackets Simple To-Do_ entry under _Applications_ section with the following properties:
```
read and write access on all your boards, read and write access on all your organizations
Approved: DATE OF APPROVAL
Never Expires
```
You may also try to _Revoke_ this token and get another token.

**Comment by chrisfrees** [comment]
I'm on Yosemite (10.10.2) running Brackets 1.2. 
I checked the Trello settings and the properties are the same.
I'll try revoking the token.
